configurator/main.py

```diff
@@ -0,0 +1,123 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# !/usr/bin/env python
+import ast
+from distutils import dir_util
+import logging
+import shutil
+from configparserutil import ConfigParserUtil
+import os
+import constants
+from jinja2 import Environment, FileSystemLoader
+
+logging.basicConfig(level=logging.DEBUG,
+                    format='%(asctime)s %(levelname)-8s %(message)s',
+                    datefmt='%a, %d %b %Y %H:%M:%S',
+                    filename=constants.LOG_FILE_LOCATION,
+                    filemode='w')
+
+PATH = os.path.dirname(os.path.abspath(__file__))
+TEMPLATE_ENVIRONMENT = Environment(
+    autoescape=False,
+    loader=FileSystemLoader(os.path.join(PATH)),
+    trim_blocks=False)
+PACK_LOCATION = None
+
+# parse the xml file and return the content as a text
+def render_template(template_filename, context):
+    return TEMPLATE_ENVIRONMENT.get_template(template_filename).render(context)
+
+
+# This method write the output generated from jinja file to xml file
+# template_path : Path to the template file
+# output_path : Path to the output xml file
+# context : dictionary containing values to be used by jinja engine
+def create_output_xml(template_path, output_path, context):
```

Comment:
The same concern, please use docstring format for method comments:
http://legacy.python.org/dev/peps/pep-0257/